src/components/DataMessenger/DataMessenger.js

```diff
@@ -819,8 +819,8 @@ export default class DataMessenger extends React.Component {
           position="bottom" // preferred position
           content={
             <div>
-              <div className="clear-messages-confirm-popover">
-                <div
+              {/*<div className="clear-messages-confirm-popover">
```

Comment:
remove

Reply:
cant remove this. It's only commented to hide from prod